src/client/pages/Download/components/DownloadPageContent.tsx

```diff
@@ -174,8 +175,9 @@ export const DownloadPageContent = ({
 const CrossSells = () => {
   const { loading, crossSells } = useCrossSells()
   const textKeys = useTextKeys()
+  const currentLocale = useCurrentLocale()
 
-  if (loading) return null
+  if (loading || !crossSells?.length) return null
```

Comment:
nit: do we need to check `loading`?

I would also make the check more explicit =>

```
if (!crossSells || crossSells.length === 0) return null
```

I just find it easier to read. We are actually covering two cases: 1. are we loading data 2. do we have 0 possible cross-sells?

Reply:
Yeah, I can change it to your suggestion. The `loading` check was added by you 🙃 
If it would only be `!crossSells?.length` we might briefly show the header "Others also bought" and then remove it if `length === 0`, but your suggestion would solve that 👍